### PR TITLE
Fix markup issues

### PR DESCRIPTION
**Pull request by spk**
Hi just fixing some markup errors found while working on https://github.com/ruby/www.ruby-lang.org/pull/1420 in case this is not merged; cheers

### CODE REVIEW

File: admin/translation-status/index.html [hunk 1]
```diff
@@ -1,6 +1,3 @@
----
----
-
```
Comment (znz) [inline admin/translation-status/index.html]: Why do you remove [YAML front matter](https://jekyllrb.com/docs/front-matter/)?

If you show the file directly, I recommend to use rake commands described in [Preview Your Changes](https://github.com/ruby/www.ruby-lang.org#preview-your-changes).

Comment (spk) [reply admin/translation-status/index.html]: Fixed in https://github.com/ruby/www.ruby-lang.org/commit/d0cbdb32665864929c618ac1c80e223af31250c3 thanks

**Comment by stomar** [comment]
@spk Thanks for catching those errrors! But please avoid silently introducing unrelated changes.

Also thanks to @znz.

**Comment by spk** [comment]
Sorry again I did not know fixed in https://github.com/ruby/www.ruby-lang.org/pull/1988 by @znz thanks

**Pull request merged**